packages/dropdowns.next/src/elements/menu/Item.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import AddIcon from '@zendeskgarden/svg-icons/src/16/plus-stroke.svg';
+import NextIcon from '@zendeskgarden/svg-icons/src/16/chevron-right-stroke.svg';
+import PreviousIcon from '@zendeskgarden/svg-icons/src/16/chevron-left-stroke.svg';
+import CheckedIcon from '@zendeskgarden/svg-icons/src/16/check-lg-stroke.svg';
+import { IItemProps, OptionType as ItemType, OPTION_TYPE } from '../../types';
+import useMenuContext from '../../context/useMenuContext';
+import { ItemContext } from '../../context/useItemContext';
+import { StyledItem, StyledItemContent, StyledItemIcon, StyledItemTypeIcon } from '../../views';
+import { ItemMeta } from './ItemMeta';
+
+const ItemComponent = forwardRef<HTMLLIElement, IItemProps>(
+  ({ children, icon, isDisabled, type, ...props }, ref) => {
+    // TODO [jz] remove state for styling demo only
+    const [isActive, setIsActive] = useState(false);
+    const [isChecked, setIsChecked] = useState(false);
+    // END TODO
```

Comment:
Could you talk more on this `TODO`? Is it intended to be removed after this PR?

Reply:
It's a TODO for @geotrev when he gets to implementing the real component :wink:

Reply:
Cool cool!